tasks/prelim.yml

```diff
@@ -139,8 +139,21 @@
   shell: "egrep ':[0-9]{4}' /etc/passwd | cut -d: -f1,4,6"
   changed_when: no
   failed_when: no
-  register: rhel7stig_passwd_homedir_check
-  when: rhel_07_020620 or rhel_07_020630 or rhel_07_020640 or rhel_07_020650
+  check_mode: yes
```

Comment:
should be "check_mode: no" which causes the command to execute, even if check_mode is otherwise in effect.  "command" and "shell"  tasks are normally skipped in check mode because ansible can't know if it will change something.  That's why travis-ci failed.